web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -5,27 +5,40 @@ import 'rxjs/add/operator/timeout';
 
 import { StoreService } from 'app/store/store.service';
 import { Settings } from 'app/settings';
-import { Account } from '@burstjs/core';
+import { Account, compose, ApiSettings } from '@burstjs/core';
 import { generateMasterKeys, Keys, encryptAES, hashSHA256, getAccountIdFromPublicKey } from '@burstjs/crypto';
 import { isBurstAddress, convertNumericIdToAddress, convertAddressToNumericId } from '@burstjs/util';
+import { environment } from 'environments/environment.prod';
 
 
 @Injectable()
 export class AccountService {
     private nodeUrl: string;
+    private api: any; //todo
```

Comment:
Todo what? Interface type? _And my axe_

Reply:
Yeah we could write a generic for the compose API that would avoid us having to update it whenever we add a new method, but also avoids this `any`.

Reply:
Humm, cannot see right now how this will work. Does the Code Completion will be supported with that method, too. I think this would be extremely useful

Reply:
Sadly it wouldn't work for type completion, but would basically typecheck against its structure. I'll see if I can do it.